content/en/docs/instrumentation/java/libraries.md

```diff
@@ -23,3 +24,164 @@ some instrumentation libraries for your dependencies standalone. To find out
 which standalone instrumentation libraries are available, take a look at
 [this list](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks).
 Follow the instructions of each instrumentation library to set them up.
+
+## Example app
+
+We will instrument an HTTP client application using library instrumentation
+which will make a call to HTTP server.
+
+You can use the dice example app as HTTP server from
+[Getting Started](/docs/instrumentation/java/getting-started/) or you can create
+your own HTTP server.
+
+### Dependencies
+
+To begin, set up an environment in a new directory called
+`java-simple-http-client`. Within that directory, create a file called
+`build.gradle.kts` with the following content:
```

Comment:
Adjust the mention of the filename here.

Reply:
Sure @chalin . Let me try something and get back.

Reply:
Hi @chalin I have added a note for Maven. Please suggest a change if needed.

Reply:
I'm ok with the (adjusted) note (see my review comment of Jan 10).
@svrnm @theletterf - WDYT?